chamber/data_transfer.py

```diff
@@ -1,11 +1,39 @@
+import pandas as pd
 from os import listdir
 from re import compile
+from nptdms import TdmsFile
+
+FILE_PATH = ""
 
 def list_tdms(file_path):
     """Returns a list of the .tdms files contained within the argument file."""
+    FILE_PATH = file_path
     file_list = []
     regex = compile(".tdms")
     for file in listdir(file_path):
         if regex.match(file, len(file) - len(".tdms")):
             file_list.append(file)
     return file_list
+
+def get_settings(tdms_obj):
+	"""returns a dictionary of the initial state of Tests in the TdmsFile object argument"""
+    df = pd.DataFrame({chan.channel:chan.data for chan in tdms_obj.group_channels('Data')})
+
+    settings = {'initial_dew_point': df.DewPoint[0], 'initial_duty': df.DutyCycle[0], 'initial_mass': df.Mass[0]
+                , 'initial_pressure': df.Pressure[0], 'initial_temp': sum([df['TC'+str(i)][0] for i in range(14)])/14
```

Comment:
We don't want to take the average of all 14 of the thermocouples. We actually only want the ones that are measuring the wall and air temps. I know that thermocouples 0-3 are reserved for the Stefan tube. Thermocouples 4-11 measure wall temps and 12-13 measure air temps. So you actually want the average of thermocouples 4-13.